setup.py

```diff
@@ -13,7 +13,7 @@ def find_dev_required():
 
 setup(
     name="district42",
-    version="1.7.0",
+    version="1.7.1",
```

Comment:
Could you please revert the changes made to the version in `setup.py`? I will update the version using the `bumpversion` command instead. Using `bumpversion` is preferable because it ensures consistency across all relevant files and automates the versioning process, reducing the risk of manual errors. Thank you!

Reply:
done :)